tests/zenko_tests/node_tests/package.json

```diff
@@ -24,6 +24,7 @@
     "test_api": "mocha --tags ${MOCHA_TAGS} --exit -t 10000 --recursive backbeat/tests/api",
     "test_retry": "mocha --tags ${MOCHA_TAGS} --exit -t 10000 --recursive backbeat/tests/retry",
     "test_crr_pause_resume": "mocha --tags ${MOCHA_TAGS} --exit -t 10000 --recursive backbeat/tests/crr-pause-resume",
+    "test_lifecycle": "mocha --tags ${MOCHA_TAGS} --exit -t 10000 --recursive backbeat/tests/lifecycle",
```

Comment:
you will also need to add this to the test entrypoint so that it is ran in the CI  
https://github.com/scality/Zenko/blob/development/1.1/tests/zenko_tests/docker-entrypoint.sh#L27

Reply:
I'm holding off on CI runs because the full feature hasn't been merged yet (I've been using these tests for test-driven-development with our development branches). I'm hoping with this PR to get high-level reviews, and then once all the feature code is merged, we'll be able to easily automate the testing in the CI to help with long-term maintenance.